theconfluenceBOT/new.py

```diff
@@ -188,7 +239,7 @@ def get_departures(reddit):
             lastpop = len(lastrundata)
 
     # Check if runs are the same
-    if lastrun_sub != lastrun_bot:
+    if lastRunNumberSub != lastrun_bot:
```

Comment:
`lastrun_bot` is a string and `lastRunNumberSub` is an integer. Don't worry I'll change this (make `lastrun_bot` an int).